sklearn/tree/_tree.pyx

```diff
@@ -2707,12 +2707,11 @@ cdef class DepthFirstTreeBuilder(TreeBuilder):
                                          split.threshold, impurity, n_node_samples,
                                          weighted_n_node_samples)
 
-                if is_leaf:
-                    # Don't store value for internal nodes
-                    splitter.node_value(tree.value +
-                                        node_id * tree.value_stride)
+                # Store value for all nodes, to facilitate tree/model inspection and interpretation
+                splitter.node_value(tree.value +
+                                    node_id * tree.value_stride)
```

Comment:
you can use a single line and still fit in less than 80 columns now that the ident level was reduced by 4 spaces:

```python
                splitter.node_value(tree.value + node_id * tree.value_stride)
```